book/website/collaboration/hybrid-collab/hybrid-collab-challenges.md

```diff
@@ -8,19 +8,26 @@ Some of the challenges faced in hybrid settings are highlighted in this chapter.
+
+The variety of communication tools used in hybrid work presents challenges for inclusiveness, especially when using asynchronous communication tools. Poorly organised hybrid events can isolate online participants, and incorporating communication tools into hybrid collaboration can be complex. Hybrid meetings require different etiquette, and unequal access to communication tools may exclude some individuals, especially those at early career stages who might still be exploring navigating such spaces.
 
 ## Lack of space to focus and be attentive
 There are disparities in facilities and services between in-person and online settings.
-Distraction comes easily with an online set up.
+Distraction comes easily with an online setup.
 When online, individuals may encounter higher levels of noise and distractions due to personal responsibilities that are not immediately apparent, such as caring for children or elderly family members. 
-In addition to the communication challenges discussed earlier, individuals may not be aware of the support resources available to them and may struggle to ask questions or access the necessary assistance.
-This is particularly problematic for individuals with caregiving responsibilities who may lack a quiet space to participate in remote conferences, workshops, trainings, and events. 
-In essence, the lack of essential facilities and support needed to participate attentively can discourage the online participants to continue attending the event online.
+In addition to the communication challenges discussed earlier, individuals may not be aware of the support resources available and may struggle to ask questions or access the necessary assistance.
+This is particularly problematic for individuals with caregiving responsibilities who may need a quieter space to participate in remote conferences, workshops, trainings, and events. 
+In essence, the lack of essential facilities and support needed to participate attentively can discourage online participants from attending the event remotely.
+
+## Coordination and scheduling
+There are also challenges in coordination and scheduling, especially across time zones. An important concern is finding common times for synchronous hybrid meetings among participants across time zones. Personal schedules and commitments further complicate this coordination. While asynchronous communication helps manage these differences by allowing participants to contribute at their convenience, it requires careful planning to ensure that all team members can participate effectively without feeling overwhelmed or excluded. Additionally, relying on asynchronous communication can impact the immediacy of real-time collaboration, slowing down engagement levels, decision-making processes, and resolving issues, as participants needed for feedback or support are not always immediately available.  
```

Comment:
```suggestion
There are also challenges in coordination and scheduling, especially across timezones. An important concern is finding common times for synchronous hybrid meetings among participants across time zones. Personal schedules and commitments further complicate this coordination. While asynchronous communication helps manage these differences by allowing participants to contribute at their convenience, it requires careful planning to ensure that all team members can participate effectively without feeling overwhelmed or excluded. Additionally, relying on asynchronous communication can impact the immediacy of real-time collaboration, slowing down engagement levels, decision-making processes, and resolving issues, as participants needed for feedback or support are not always immediately available.  
```

Reply:
Might have gotten this one wrong re: spelling of timezones! We also use the spelling 'timezone a lot' as opposed to time zone. Woops!

Reply:
Looking at it again, I think I prefer "timezones" as opposed to "time zones".